pyproject.toml

```diff
@@ -1,49 +1,50 @@
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["hatchling>=1.20", "hatch-vcs>=0.4"]
+build-backend = "hatchling.build"
 
-[tool.poetry]
-name = "st_package_reviewer"
-version = "0.4.0-dev"
+[project]
+name = "st-package-reviewer"
+dynamic = ["version"]
 description = "Review packages for Sublime Text."
-authors = ["FichteFoll <fichtefoll2@googlemail.com>"]
-license = "MIT"
 readme = "README.md"
-homepage = "https://github.com/packagecontrol/st_package_reviewer"
-repository = "https://github.com/packagecontrol/st_package_reviewer"
-packages = [
-    { include = "st_package_reviewer" },
-    { include = "tests", format = "sdist" },
-]
+license = { text = "MIT" }
+authors = [{ name = "FichteFoll", email = "fichtefoll2@googlemail.com" }]
+requires-python = ">=3.13,<4.0"
 classifiers = [
-    'Development Status :: 4 - Beta',
-    'Environment :: Console',
-    'Intended Audience :: Developers',
-    'License :: OSI Approved :: MIT License',
-    'Operating System :: OS Independent',
-    'Topic :: Software Development :: Quality Assurance',
+  "Development Status :: 4 - Beta",
+  "Environment :: Console",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.13",
+  "Topic :: Software Development :: Quality Assurance",
+]
+dependencies = [
+  "github3.py>=4,<5",
+  "pyperclip>=1.7,<2",
 ]
 
-[tool.poetry.scripts]
-st_package_reviewer = "st_package_reviewer.__main__:main"
-
-[tool.poetry.dependencies]
-"github3.py" = "^4.0.0"
-pyperclip = "^1.7.0"
-python = "^3.8.1"
-
-[tool.poetry.group.lint.dependencies]
-flake8 = "^7.1.1"
-flake8-docstrings = "^1.7.0"
+[project.optional-dependencies]
+dev = [
```

Comment:
uv docs recommend to use dependency groups, not "optional dependencies" for dev deps: https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies